### ISSUE TITLE
ICS4: Wrong inequality for timeoutPacket 

### ISSUE THREAD

**Issue by mpoke** [cosmos/ibc#827]
In the pseudocode for the `timeoutPacket` method, see https://github.com/cosmos/ibc/blob/main/spec/core/ics-004-channel-and-packet-semantics/README.md#sending-end, the following code has the wrong inequality: 
```typescript
abortTransactionUnless(
      (packet.timeoutHeight > 0 && proofHeight >= packet.timeoutHeight) ||
      (packet.timeoutTimestamp > 0 && connection.getTimestampAtHeight(proofHeight) > packet.timeoutTimestamp))
```
It should be `>=` instead of `>`, i.e.,
```typescript
abortTransactionUnless(
      (packet.timeoutHeight > 0 && proofHeight >= packet.timeoutHeight) ||
      (packet.timeoutTimestamp > 0 && connection.getTimestampAtHeight(proofHeight) >= packet.timeoutTimestamp))
```
This is cf. the spec for receiving packets (see https://github.com/cosmos/ibc/blob/main/spec/core/ics-004-channel-and-packet-semantics/README.md#receiving-packets) and the [ibc-go implementation](https://github.com/cosmos/ibc-go/blob/f8f879b7bd3c5e14c5c334f703462cbc51f34ad0/modules/core/04-channel/keeper/timeout.go#L71).

**Comment by cwgoes** [comment]
oh wow, good catch. glad that ibc-go implemented the right thing...

**Comment by mpoke** [comment]
Yeah, this was caught during the Interchain Security protocol audit with the Apalache team. Thanks @Kukovec

**Comment by gjermundgaraba** [comment]
This issue was mistakenly reopened during some integration work, so closing again now.
